src/modules/useSupersetEmbeddedDashboardFieldsState.js

```diff
@@ -0,0 +1,94 @@
+import { useCallback, useReducer } from 'react'
+
+export const fieldNames = {
```

Comment:
Some years ago we decided that exported constants should not be contained in an object since it can be error prone. For example if the consumer misspells the prop name, it can go undetected.

Suggest instead to export each of these as a separate constant.